docs/csharp/getting-started/index.md

```diff
@@ -21,7 +21,7 @@ The following topics are available:
 
 - [Building a C# Hello World application with .NET Core in Visual Studio 2017](../../core/tutorials/with-visual-studio.md)
 
-   Visual Studio 2017, the latest release of Visual Studio, lets you code, compile, run, debug, profile, and publish your applications from an integrated development environment for Windows.
+   Visual Studio 2017, lets you code, compile, run, debug, profile, and publish your applications from an integrated development environment for Windows.
```

Comment:
At this point, we should just update this to Visual Studio 2019:

```suggestion
   Visual Studio 2019, lets you code, compile, run, debug, profile, and publish your applications from an integrated development environment for Windows of Mac.
```

Reply:
It was the first option came to my mind, but the previous line has a link to article: `Building a C# Hello World application with .NET Core in Visual Studio 2017`

I think it could be confusing a little bit ?

Reply:
Good point.

Adding @mairaw and @Thraka The linked article is under the .NET Core section. I think we're planning to update that one as well during this wave. Is that correct?

Reply:
Yes, I'm planning to update that article soon.

Reply:
@Youssef1313 Let's accept the current suggestion, and it will reconcile correctly in a few days.

Thanks for making this update.

Reply:
Edited the comment to say Windows OR Mac.. it said Windows OF Mac. 😄

Reply:
@BillWagner, I applied it.
@Thraka , I didn't even notice that addition in suggestion, thought the change was only 2017 => 2019. Thanks for correcting it :)